crates/kitsune_p2p/kitsune_p2p/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## \[Unreleased\]
 
+- *BREAKING* Updates tx5 to a version using new endpoint state logic and a new incompatible protocol. [\#3287](https://github.com/holochain/holochain/pull/3287)
```

Comment:
👍🏻 Is there going to be a back-port for this or is this just on 0.3?

Reply:
The plan right now is just on 0.3.